config/rbac/flowtracing_config_subscriber_role.yaml

```diff
@@ -0,0 +1,32 @@
+kind: Role
```

Comment:
I wonder if this is better placed here or in the CNO repo.. is it something discussed already with the cno team?

Reply:
No, I haven't. My reasoning was "NOO is the owner/manager of this resource, so NOO is the responsible of managing the access to it".

I'm also thinking in the namespace. If we change it to e.g. `openshift-network-observability` in the upstream version, we shouldn't need to do the change in the CNO.

Reply:
Thinking again, you are right, it must rely in the CNO namespace to make it independent from any change in the upstream/downstream namespace, so we should implement it as part of https://issues.redhat.com/browse/NETOBSERV-66

Reply:
do you think we need to add a field in our CR for the CNO namespace?

Reply:
Yes, as an optional argument that defaults to `openshift-network-operator`